Content.Server/Cargo/Systems/CargoSystem.Bounty.cs

```diff
@@ -314,7 +327,7 @@ private HashSet<EntityUid> GetBountyEntities(EntityUid uid)
                 var children = GetBountyEntities(ent);
                 foreach (var child in children)
                 {
-                 entities.Add(child);
+                    entities.Add(child);
```

Comment:
Im now going to nitpick.
```suggestion
                 entities.Add(child);
```